packages/components/src/Tokens/Tokens.tsx

```diff
@@ -44,15 +43,21 @@ export class UserTokenAccount extends React.Component<UserTokenAccountProps, Use
   }
 
   public render(): JSX.Element | null {
-    const { user, addWalletPath } = this.props;
-    const accountSignupStep = this.getUserStep(user);
+    const { user, addWalletPath, network, foundationAddress, faqUrl, supportEmailAddress } = this.props;
+    const { isTutorialModalOpen } = this.state;
 
-    const loggedInState = accountSignupStep ? TOKEN_PROGRESS.COMPLETED : TOKEN_PROGRESS.ACTIVE;
-    const tutorialState = this.props.userTutorialComplete ? TOKEN_PROGRESS.COMPLETED : TOKEN_PROGRESS.ACTIVE;
+    const accountSignupComplete = this.getAccountComplete(user);
+    const tutorialComplete = this.getTutorialComplete(user);
+    const userAccount = this.getUserAccount(user);
 
-    // TODO:Sarah - commented out for testing
-    // const buyState =
-    //  accountSignupStep && this.props.userTutorialComplete ? TOKEN_PROGRESS.ACTIVE : TOKEN_PROGRESS.DISABLED;
+    const loggedInState = accountSignupComplete ? TOKEN_PROGRESS.COMPLETED : TOKEN_PROGRESS.ACTIVE;
+    const tutorialState =
```

Comment:
Minor suggestion, but the super nested ternaries can be a bit hard to read. `if / else if` or `switch` statements are more verbose, but quicker to understand for someone unfamiliar with the code